### ISSUE TITLE
Drop Down box for Navbar on Homepage

### ISSUE THREAD

**Issue by uniiquecornnx** [Shardeum/shardeum-docs#260]
If we could add a drop-down on the Home Page for the nav bar items including 'Connect to Shardeum', 'Claim Testnet SHM' etc., It would look more representable.

<img width="959" alt="Screenshot 2023-10-17 142251" src="https://github.com/Shardeum/shardeum-docs/assets/116641912/2df8ba20-f1c2-476e-a6a2-04aa1bb97fd6">

**Comment by skundu42** [comment]
Can you share some more details on the suggestion? Do you mean to add a  drop-down nav bar with the same links?

**Comment by uniiquecornnx** [comment]
Yes exactly, add a drop down with the same links, to make it look more organised. Like the drop-down does appear when we smaller the size of the chrome but it disappears when opened in full screen, and all the links are visible on the same line making it a bit confusing for the user.

**Comment by skundu42** [comment]
I am not sure if I am getting your idea. Can you share a reference image/website which has similar functionality?
